evaluate/metrics/metrics.go

```diff
@@ -70,11 +75,17 @@ func (m Metrics) String() string {
 
 // StringCSV returns a CSV row string representation of the metrics.
 func (m Metrics) StringCSV() []string {
+	assessment := m.Assessments
+	if assessment == nil {
+		assessment = Assessments{}
+	}
+
 	return []string{
 		fmt.Sprintf("%d", m.Total),
 		fmt.Sprintf("%d", m.Executed),
 		fmt.Sprintf("%d", m.Problems),
 		fmt.Sprintf("%.0f", m.AverageCoverage()),
+		fmt.Sprintf("%d", assessment[AssessmentKeyNoExplanations]),
```

Comment:
Make this generic. Should be a function that counts all.

Also follow-up: all the metrics we have here should be in the assessment at one point. so would be good to print the whole map right away?